### DISCUSSION TITLE
Welcome to FrACT10 Discussions!

### DISCUSSION THREAD

**Discussion by michaelbach** [michaelbach/FrACT10#10] [Announcements]
## 👋 Welcome!
  Let's use _Discussions_ as a place to connect and play with ideas where to go with FrACT.
  * Please ask questions you’re wondering about
  * Share ideas
  * Engage
  * Welcome others and are open-minded.

**Comment by leulx1** [comment]
Hi Michael,

First of all, thank you for creating and maintaining FrACT—it's a great tool.

I have a feature request. Would it be possible to add a number-based eye test (digits 0–9) as an optotype option? In some situations, using numbers could be more practical or familiar for participants than letters or Landolt Cs.

Is this something you would consider adding in a future release?

Thanks for your time and for all your work on FrACT!

**Reply by michaelbach** [reply]
Dear Leul:
Thanks for your kind comment. Digits, hmm. I had already looked into this a while ago and found a number of concerns:
• There is no agreed/standardized character set for digits to my knowledge.
• All letters are drawn with vector graphics adhering to the Sloan shapes. For numbers I would not like to use a font – too little control, and I would have to come up in some way with correct vector shapes.
• The outcome of digits is better acuity by 0.13 LogMAR, at least for the charts we used (I have a paper on this, unfortunately in German; Rohrschneider, Spittler & Bach 2019). And that certainly depends on the digit shapes.
So, all things considered, this does not have a high priority for me. Of course, were you to have good solutions for the above problems… 
Best, Michael
